sdk/openai/Azure.AI.OpenAI/tests/Samples/00_ClientConfiguration.cs

```diff
@@ -35,7 +36,27 @@ public void CreateAnAzureOpenAIClientWithEntra()
         AzureOpenAIClient azureClient = new(
             new Uri("https://your-azure-openai-resource.com"),
             new DefaultAzureCredential());
-        ChatClient chatClient = azureClient.GetChatClient("my-gpt-35-turbo-deployment");
+        ChatClient chatClient = azureClient.GetChatClient("my-gpt-4o-mini-deployment");
+        #endregion
+    }
+
+    public void UseAzureGovernment()
+    {
+        #region Snippet:ConfigureClient:GovernmentAudience
+        AzureOpenAIClientOptions options = new()
+        {
+            Audience = AzureOpenAIAudience.AzureGovernment,
+        };
+        AzureOpenAIClient azureClient = new(
+            new Uri("https://your-azure-openai-resource.com"),
+            new DefaultAzureCredential());
+        ChatClient chatClient = azureClient.GetChatClient("my-gpt-4o-mini-deployment");
+        #endregion
+        #region Snippet:ConfigureClient:CustomAudience
+        AzureOpenAIClientOptions optionsWithCustomAudience = new()
```

Comment:
Should this custom audience be added as a separate sample? It could be a little confusing otherwise.